Description: Remove git from dzil config
 Git config is suitable for upstream, but does not work
 when building the package
Forwarded: not-needed
Author: Dod
--- a/dist.ini
+++ b/dist.ini
@@ -18,18 +18,6 @@
 [NextRelease]
 format = %v%T %{yyyy-MM-dd}d
 
-; use 'V=2.234 dzil release' to override version number
-[Git::NextVersion]
-
-[Git::Check]
-allow_dirty = dist.ini
-allow_dirty = Changes
-
-[Git::Commit]
-
-[Git::Tag]
-[Git::Push]
-
 [GatherFile]
 filename = debian/dh/dh_cme_upgrade
 prefix = bin
@@ -71,6 +59,3 @@
 [PodWeaver]
 
 [Prepender]
-
-;-- see https://metacpan.org/pod/Dist::Zilla::Plugin::Signature
-[Signature]
